### ISSUE TITLE
NTR: Glucosidase I deficiency (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8177]
New term request Glucosidase I deficiency (segal_040409201713):   UMLS: 1853739

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000576
- label: Reduced tissue mannosyl-oligosaccharide glucosidase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:10788335
- def: Activity of mannosyl-oligosaccharide glucosidase (MOGS) in tissues below the lower limit of normal. MOGS activity can be measured in multiple tissues including liver and cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8177
- synonyms:
